src/components/Home.js

```diff
@@ -23,11 +24,11 @@ class Home extends Component<{}, State> {
   }
 
   componentDidMount() {
-    // there will likely be a better/ additional deciding factor here
-    // once integrated w/ swarm
-    if (localStorage.getItem('notes')) {
-      this.setState({ initial: false })
-    }
+    getNotes().then(result => {
```

Comment:
Rather than having this logic here, maybe this could be provided as a prop by the `App` component, considering it's the one storing the notes? Something like `<Home initial={this.state.notes.length === 0} />`.